src/test/migrations/index.ts

```diff
@@ -25,6 +25,7 @@ describe(`Migrations:`, () => {
         .filter((migration: string) => !ranMigrations.includes(migration))
         .sort()
         .map(async (file: string) => {
+          // eslint-disable-next-line no-console
```

Comment:
Can we use Logger here instead of console.log ?

Reply:
Does logger work in this context? Can you check?